common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,7 +1238,11 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
-                msg = _(u"Error: {msg}").format(msg=inst.message)
+                if type(inst) == StudentInputError:
+                    msg = _(u"{msg}").format(msg=inst.message)
+                else:
+                    msg = _(u"Error: {msg}").format(msg=inst.message)
```

Comment:
Note:
I did not want to remove the Error text for messages beyond Student input errors.  There are a number of possible error messages in Capa that get displayed here.